### PR TITLE
[NUI] add enum validation for Grid,FlexLayout

### PR DESCRIPTION
**Pull request by dasg34**
This validation check is added by TCT reviewer request.

### API Changes ###

 - ACR:

### CODE REVIEW

**Review by woohyun — APPROVED** [review]
This looks good to me. Thanks.

**Comment by woohyun** [comment]
ready to merge

File: src/Tizen.NUI/src/public/Layouting/FlexLayout.cs [hunk 1]
```diff
@@ -473,6 +477,8 @@ public FlexWrapType WrapType
             }
             set
             {
+                if (value != FlexWrapType.NoWrap || value != FlexWrapType.Wrap)
```
Comment (dongsug-song) [inline src/Tizen.NUI/src/public/Layouting/FlexLayout.cs:480]: could you add excepton comment here? <exception cref=xxx> xxx </exception>
ex) https://github.com/Samsung/TizenFX/blob/master/src/Tizen.NUI/src/public/Window.cs#L69

File: src/Tizen.NUI/src/public/Layouting/FlexLayout.cs [hunk 2]
```diff
@@ -509,6 +517,8 @@ public AlignmentType ItemsAlignment
             }
             set
             {
+                if (value < AlignmentType.Auto || value > AlignmentType.Stretch)
+                    throw new ArgumentException(nameof(ItemsAlignment));
```
Comment (dongsug-song) [inline src/Tizen.NUI/src/public/Layouting/FlexLayout.cs:521]: same as above

File: src/Tizen.NUI/src/public/Layouting/GridLayout.cs [hunk 3]
```diff
@@ -283,6 +283,9 @@ public Orientation GridOrientation
             set
             {
                 if (gridOrientation == value) return;
+                if (value != Orientation.Horizontal || value != Orientation.Vertical)
```
Comment (dongsug-song) [inline src/Tizen.NUI/src/public/Layouting/GridLayout.cs:286]: same as above

File: src/Tizen.NUI/src/public/Layouting/LayoutGroup.cs [hunk 4]
```diff
@@ -562,5 +562,15 @@ internal static void OnChildPropertyChanged(Binding.BindableObject bindable, obj
...
+        {
+
+            return (Binding.BindableObject bindable, object value) =>
+            {
+                int @enum = (int)value;
+                return enumMin <= @enum && @enum <= enumMax;
+            };
+        }
     }
 }
```
Comment (dongsug-song) [inline src/Tizen.NUI/src/public/Layouting/LayoutGroup.cs:576]: could you add new line at the end of file?
reference) https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline

**Comment by dongsug-song** [comment]
@dasg34 
could you resolve "conflict" ?
thank you.

**Pull request merged**